main.go

```diff
@@ -37,7 +41,21 @@ func main() {
 	if os.Getenv("NAMESPACE") != "" {
 		*namespace = os.Getenv("NAMESPACE")
 	}
-	if err := operator.Run(ctx, *namespace); err != nil {
+
+	var ns []string
+	if os.Getenv("WATCH_NAMESPACE") != "" {
+		*namespaces = os.Getenv("WATCH_NAMESPACE")
+	}
+
+	if *namespaces != "" {
+		ns = strings.Split(*namespaces, ",")
+	}
+
+	//TODO check the proper namespace configuration, should it be the same namespace as the rancheros-operator?
```

Comment:
I guess this TODO is obsolete, I think the current state is flexible and clear enough about namespaces